### PR TITLE
Create Anya subfolder and initial custom creation

### PR DESCRIPTION
**Pull request by anyahush**
**Create Anya subfolder and initial custom creation*

## PR contains:
- Create Anya subfolder with index and CSS pages
- Create initial set up of pages
- Create custom CSS animation creation

## Hacktoberfest 2022:
- [x] Does this PR follow the `intermediate.md` guide and submitted as part of Hacktoberfest 2022?
- [x] Yes

## Breaking changes
- None

## Screenshots
- None

### CODE REVIEW

File: assets/data/community.json
Comment (Aramantos) [inline assets/data/community.json:391]: Hi Anya,

There is a conflict with merging this pull request.
You will need to resolve this before we can merge your fork with the main repo.

If you have any questions feel free to reach out.
Thank you

**Review by Aramantos — CHANGES_REQUESTED** [review]
Please see the comments left on the conflict error
Any questions, please feel free to reach out

File: assets/data/community.json
Comment (anyahush) [reply assets/data/community.json:391]: Hi, Okay I will have a look. Thank you

Comment (auxfuse) [reply assets/data/community.json:391]: @anyahush the problem isn't a merge conflict here, it's that you have removed "NiclasTanskanen"'s details from the community.json file and replaced them with your own 😄 I've attached a screenshot of the removal, and how it should look like.

What was removed:
image

How it should be:
image

Can you put those details back in please? and then add your own object following the guidelines in the [basic.md](https://github.com/auxfuse/hackathon-git-labs/blob/main/basic.md) file. 🤜 Everything else in this is great, we just need to make sure that the `community.json`  file is handled correctly.

When this is done, add, commit, and push back to the branch associated with this PR (in this case, you are on the main branch of your fork, we recommend using branches for subsequent PRs as per the documentation for this project) to auto-update this PR and let us know via comment. We will review, and then will resolve this conversation too pending approval & merging. 🎉

**Comment by auxfuse** [comment]
@anyahush @Aramantos I've added to the requested changes 🤜 Let me know if there is anything you need help with and I can jump in on my phone 😄 Happy Sunday lads!

File: assets/data/community.json
Comment (anyahush) [reply assets/data/community.json]: So sorry about that!! Silly me 🙈 I have fixed the error. Thanks for your help 😊

**Review by auxfuse — APPROVED** [review]
Thanks a million @anyahush great work! LGTM 🎉🎉🎉🎉

**Pull request merged**
